hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -422,3 +422,18 @@ object JSBuilder:
 end JSBuilder
 
 
+trait JSBuilderSanityChecks(val instrument: Bool) extends JSBuilder:
```

Comment:
No reason to make this a `val`. Doing so will prevent other similar traits (copied from this model) from defining their own `instrument` parameter.